### PR TITLE
Faq

### PR DESCRIPTION
**Pull request by cpayton-source**
For this branch, I created a basic layout for a FAQ(frequently asked questions page) where the user would click on a button depending on which questions they would like answered.

### CODE REVIEW

File: App/src/main/webapp/script.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+function state(){
+    const state = 'Just califonia for now.';
```
Comment (jacinda22) [inline App/src/main/webapp/script.js]: Should be California instead of califonia

**Review by jacinda22 — APPROVED** [review]

File: App/src/main/webapp/script.js (hunk 1 above)
Comment (cpayton-source) [reply App/src/main/webapp/script.js]: Thanks I didn't catch that I'll update now

**Review by perrycate — APPROVED** [review]
Looks good to me! I like the small commits. ;)

**Comment by perrycate** [comment]
Doesn't have to be in this PR, but I would suggest that you consider making the styling match the rest of the site, if you have time.

**Pull request merged**